.gitignore

```diff
@@ -1,2 +1,5 @@
 build
-.vscode
\ No newline at end of file
+.vscode
+docs/build.md
```

Comment:
Why would you gitignoe that documentation flle?

Reply:
I seem to have added it by accident. Thank you for pointing that out. Will change that. :)